client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2554,4 +2554,34 @@ Undocumented.prototype.getAtomicSiteMediaViaProxyRetry = function (
 	return request();
 };
 
+/**
+ * Request all Licensing Partners and their keys for the current WPCOM user.
+ *
+ * @returns {Promise} A promise
+ */
+Undocumented.prototype.getJetpackLicensingPartners = function () {
```

Comment:
Of note, this was a bit confusing to me. I wasn't sure why we would want to get all partners instead of just the current partner. But, it looks like a WPCOM user to partners is a one-to-many relationship. Perhaps we could/should use an endpoint to just get all licenses?

Reply:
> instead of just the current partner

The endpoint returns all partners for the current user, if that user has multiple partners. In most (future) cases this should return a single partner with a single key and I've updated the PR to auto-select it if that is the case. For a12s, it is much more likely that multiple keys will exist so a minimal list of keys shows up for the user to select which key to use the portal with. Again, this is a minimal implementation to simplify the development process and it does not include a dropdown to change the current key, for example.

> Perhaps we could/should use an endpoint to just get all licenses?

This may not be desirable as we would be mixing licenses issued by non-billable and billable keys, for example (we could address this via tagging licenses based on their key and adding filters etc. but we haven't discussed this).